tests/e2e/util.go

```diff
@@ -0,0 +1,446 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+	cnstypes "github.com/vmware/govmomi/cns/types"
+	"github.com/vmware/govmomi/find"
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/govmomi/property"
+	"github.com/vmware/govmomi/vim25/mo"
+
+	vim25types "github.com/vmware/govmomi/vim25/types"
```

Comment:
can we bring all the `govmomi` imports together? There's another one down below. It's nicer to group them all together.